dhcp_server/config.json

```diff
@@ -1,13 +1,19 @@
 {
   "name": "DHCP server",
-  "version": "1.2",
+  "version": "1.3",
   "slug": "dhcp_server",
-  "description": "A simple DHCP server",
+  "description": "A simple DHCP server with failover capability",
   "url": "https://home-assistant.io/addons/dhcp_server/",
   "arch": ["armhf", "armv7", "aarch64", "amd64", "i386"],
   "advanced": true,
   "startup": "system",
   "host_network": true,
+  "ports": {
+    "647/tcp": 647
+  },
+  "ports_description": {
+    "647/tcp": "DHCP failover handshake"
+  },
```

Comment:
This makes no sense if you not read this data later with bashio and adjust the port based on the user settings